www/js/services.js

```diff
@@ -0,0 +1,38 @@
+angular.module('open311.services', [])
+.factory('API', function($http, $httpParamSerializer, $q) {
+
+    var recentCasesData = [];
+
+    var getRequests = function(lat, long) {
+      var deferred = $q.defer();
+
+      var params = $httpParamSerializer({
+          lat: lat,
+          long: long 
+      });
+
+      $http.get('https://seeclickfix.com/open311/v2/requests.json?' + params)
+        .then(function(response) {
+          recentCasesData = response.data;
+          deferred.resolve(response);
+        }, function(response) {
+          deferred.reject(response);
+        });
```

Comment:
I prefer using `.catch()` instead of the second parameter of `.then()`, it's more readable.

```js
$http.get('/requests.json')
     .then(function(response) {
     // Do something with the response
     })
     .catch(function(err) {
     // Handle any errors from the API
     });
```